site/_includes/layouts/reference-landing.njk

```diff
@@ -32,11 +32,11 @@
 
   <h2 id="platform_apps_apis">Platform Apps APIs</h2>
   <p>
-    These APIs support Chrome Apps on all platforms.
+    These APIs support Chrome Apps.
   </p>
   <div class="aside aside--warning">
-    These APIs are deprecated.
-    Chrome will be removing support for Chrome Apps on all platforms.
+    These APIs were deprecated in 2020. They are
+    supported only for ChromeOS until at least Jan 2025.
```

Comment:
Sorry missed this, but should we include 
`It remains supported for Enterprise and Education customers on ChromeOS until at least Jan 2025.` in this part also?

Reply:
Done.